modules/apps/frontend-js/frontend-js-translation-web/src/main/resources/META-INF/resources/TranslationModal.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayButtonWithIcon} from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import {ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayManagementToolbar from '@clayui/management-toolbar';
+import ClayModal from '@clayui/modal';
+import ClayTable from '@clayui/table';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const TranslationModal = ({
+	ariaLabels = {
+		default: Liferay.Language.get('default'),
+		manageTranslations: Liferay.Language.get('manage-translations'),
+		translated: Liferay.Language.get('translated'),
+		untranslated: Liferay.Language.get('untranslated'),
+	},
+	defaultLocale,
+	locales,
+	observer,
+	translations,
+}) => {
+	const [active, setActive] = React.useState(false);
+	const [icon, setIcon] = React.useState('search');
+	const inputRef = React.createRef();
+	const [visibleLocales, setVisibleLocales] = React.useState(locales);
+
+	return (
+		<ClayModal observer={observer}>
+			<ClayModal.Header>
+				{Liferay.Language.get('manage-translations')}
+			</ClayModal.Header>
+
+			<ClayModal.Body scrollable>
+				<ClayManagementToolbar aria-label="Management toolbar">
+					<ClayManagementToolbar.Search showMobile={true}>
+						<ClayInput.Group>
+							<ClayInput.GroupItem>
+								<ClayInput
+									aria-label={Liferay.Language.get('search')}
+									className="form-control input-group-inset input-group-inset-after"
+									onInput={() => {
+										const value = inputRef.current?.value;
+
+										const icon = value ? 'times' : 'search';
+										setIcon(icon);
+
+										if (!value) {
+											setVisibleLocales(locales);
+
+											return;
+										}
+
+										const result = Object.values(
+											visibleLocales
+										).filter((locale) => {
+											return locale.label.match(
+												new RegExp(value, 'i')
+											);
+										});
+
+										setVisibleLocales(result);
+									}}
+									placeholder={Liferay.Language.get('search')}
+									ref={inputRef}
+									type="text"
```

Comment:
I believe this is already the default "type" on the component, so there is no need to specify it (although I guess it doesn't necessarily hurt to leave it).

Reply:
You're right, if it's not needed it doesn't need to be there.

Reply:
Done in [`a0b4d93`](https://github.com/liferay-frontend/liferay-portal/pull/1184/commits/a0b4d93f9e32601df6cca2be04b9977a309b6c87)